packages/shared/src/data/spells.json

```diff
@@ -165,9 +165,6 @@
   "11327": {
     "type": "buffs_defensive"
   },
-  "12042": {
```

Comment:
were these manual changes or did you run the refresh script? i'm checking whether the removals are ok

Reply:
from the refresh script